src/org/exist/util/serializer/SAXToReceiver.java

```diff
@@ -34,9 +34,15 @@
 
     private Receiver receiver;
     private boolean inCDATASection = false;
+    private boolean sendWStoCharacters = false;
```

Comment:
make consistent with "redirectIgnorableWhitespace" ? functionally it is the same...

i'd probably prefer a shorter name, avoiding "Characters". Maybe the 'invertway' notation is a bit more clear, like "ignoreWhitespace" (normally it is not)